src/platform/auth.ts

```diff
@@ -20,6 +20,8 @@ const {
 
 const SUDO_TOKEN_VALIDITY = 20 * 60 * 1000;
 
+const USERNAME_BLACKLIST = ['root'];
```

Comment:
Err, the single openBalena user gets username `root`.

Reply:
Ah… That is going to cause issues (with ssh). Is it easily changed?